vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,75 @@
+# Entando Glossary
+
+## Entando Terms
+
+### Core Products
+
+| Term |  Description
+|:--|:--
+| Entando Platform | Comprises the complete set of software used to build Entando solutions, applications, sites or portals |
+
+### Solutions
```

Comment:
Remove Solutions and Consulting Packages altogether. I don't think those are needed in a glossary since they are product/services specific.